infrastructure/src/main/java/org/corfudb/infrastructure/ServerHandshakeHandler.java

```diff
@@ -145,7 +136,7 @@ public void channelRead(ChannelHandlerContext ctx, Object m) throws Exception {
      */
     @Override
     public void channelActive(final ChannelHandlerContext ctx) throws Exception {
-        log.info("channelActive: Incoming connection established from: {} Start Read Timeout.",
+        log.trace("channelActive: Incoming connection established from: {} Start Read Timeout.",
```

Comment:
This log statement I believe is also relevant when we're not sure if the connection from one client to a server is established. It is useful, I think we should leave it as INFO or DEBUG.

Reply:
I think one log line per connection establish is not too many, this tells us when the connection begins, and help to trace the life cycle of a connection when debugging.

Reply:
+1 we have used this message to debug client connection issues.

Reply:
Ok, if a majority of people believe we need let's change it back